.github/workflows/publish.yml

```diff
@@ -57,6 +66,12 @@ jobs:
           push: true
           build-args: |
             KUBEBENCH_VERSION=${{ steps.get_version.outputs.version }}
+            KUBECTL_VERSION=${{ env.KUBECTL_VERSION }}
+            KUBECTL_SUM_arm64=${{ env.KUBECTL_SUM_arm64 }}
+            KUBECTL_SUM_amd64=${{ env.KUBECTL_SUM_amd64 }}
+            KUBECTL_SUM_s390x=${{ env.KUBECTL_SUM_s390x }}
+            KUBECTL_SUM_ppc64le=${{ env.KUBECTL_SUM_ppc64le }}
+            TARGETARCH=${{ env.ARCH }}
```

Comment:
```suggestion
```

The `TARGETARCH` argument will be setting by auto when build multi-arch images with `docker/build-push-action`.  If we setting it in there, maybe it will cause all images to use same arch when download the kubectl binary? 
You can test it on local via `docker buildx build --platform linux/amd64,linux/arm64,linux/ppc64le,linux/s390x -t test .`.